test/generate-json.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import time
+import os
+import struct
+
+# PingMessage is generated by this project
+import sys
+sys.path.append("../src/protocol/python")
+import PingMessage as Message
```

Comment:
this looks wrong...
https://www.python.org/dev/peps/pep-0008/#package-and-module-names
"Modules should have short, all-lowercase names. Underscores can be used in the module name if it improves readability. Python packages should also have short, all-lowercase names, although the use of underscores is discouraged."

Reply:
@jaxxzer ^

Reply:
Noted: https://github.com/bluerobotics/ping-python/issues/18

will address subsequently.